src/main/java/me/coley/recaf/ui/controls/pane/SysInfoPane.java

```diff
@@ -82,7 +82,7 @@ public SysInfoPane() {
 			clip.setContent(content);
 		}), new ActionButton(translate("ui.about.opendir"), () -> {
 			try {
-				Desktop.getDesktop().open(Recaf.getDirectory().toFile());
+				UiUtil.showDocument(Recaf.getDirectory().toUri());
```

Comment:
So just to confirm, this retains the ability to open directories like the config directory, in addition to URLs?

I'm looking at `showDocument()` and it looks like its for browsers only.

Reply:
Yes, it can open files